### ISSUE TITLE
3 one-third width columns don't fit

### ISSUE THREAD

**Issue by dld47** [JoryHogeveen/genesis-widget-column-classes#13]
## Describe the bug
On the StudioPress Business Pro Theme I set three sections in a front page widget to "one third" width. Two appear correctly, but the third is displayed below them instead of next to them. If I make them one quarter they fit, but they're too narrow.

## Steps to reproduce the behavior:
1. Go to '.Front Page Widget 2
2. Click on 'Icon
3. Scroll down to Width
4. Set to one third
5. Do the same for two more icon blocks

## Expected behavior
Three icons side by side with text

## Screenshots
![image](https://user-images.githubusercontent.com/71578794/93694008-ce963400-fabb-11ea-8332-3f2937a31392.png)

## Specifications

  - Plugin Version:
  - WordPress Version:
  - Other plugins installed:
  - Theme:
  - Browser:

**Comment by JoryHogeveen** [comment]
Hello @dld47 

You forgot to check the `first` checkbox on the first widget!

Cheers, Jory

**Comment by dld47** [comment]
Jory -- thanks so much! I'd struggled with that for hours!

My last big problem: I've selected a header image and a static landing page
for www.daglowconsulting.com but the landing page is not displayed, just
the front page widget contents. The header display is all messed up, and a
large image I loaded in front page widget 1 trying a workaround shows on
top of something else, but reduced in size.

Any ideas on what page, widget or box I'm missing?

Thanks so much for your time!

Don


Don Daglow
Sr. Director for Strategic Partnerships, The Strong  www.museumofplay.org
President, Academy of Interactive Arts & Sciences Foundation
Author, Indie Games: From Dream to Delivery
<https://www.amazon.com/Indie-Games-Delivery-Don-Daglow/dp/0996781552/ref=tmm_pap_swatch_0?_encoding=UTF8&qid=&sr=>
 and The Fog Seller
<https://www.amazon.com/Fog-Seller-Mystery-Neverwinter-NightsTM-ebook/dp/B01867TXOC/ref=zg_bs_7588852011_29?_encoding=UTF8&psc=1&refRID=D8DADMSFCBA39N536H0A>

PO Box 1035
Sausalito, CA 94965                          Cell: 415-309-1579
            Skype: don.daglow
